bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -66,6 +67,7 @@
 		<xs:attribute name="required" type="xs:boolean" />
 		<xs:attribute name="readOnly" type="xs:boolean" />
 		<xs:attribute name="multiple" type="xs:boolean" />
+		<xs:attribute name="unit" type="config-description:unitType" />
```

Comment:
Yes, it definitely is convenient for the developer, we should just make sure that we remember which places we have to maintain, if we add new units or do changes to the list.
But I am ok with keeping the enum for now.